src/core/checkpoint_manager.py

```
@@ -52,8 +52,7 @@ def save_checkpoint(self, country_code: str, location_index: int, total_location
             "checkpoint_file": str(checkpoint_file),
             "location_index": location_index,
             "timestamp": checkpoint_data["timestamp"],
-            "total_locations": total_locations,
-            "measurements_count": None  # Can be updated later
+            "total_locations": total_locations
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)